internal-services/catalog/iib-add-fbc-fragment-to-index-image-task.yaml

```diff
@@ -83,49 +83,75 @@ spec:
       script: |
         #!/usr/bin/env bash
         #
+        # shellcheck disable=SC2207,SC2128,SC2178 # requires refactoring to fix these errors
```

Comment:
what even are these? Can you add a little more detail

Reply:
I removed the exclusion. Crossed fingers the code is still working (so far in the tests the seem to... but, lets hope to see in stage).